docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -17,7 +17,7 @@ By the end, you should understand how to:
 
 Here's what the final product looks like—click on the **< >** icon underneath the demo to see the full source code:
 
-{{"demo": "LoginFinal.js", "hideToolbar": false, "bg": true}}
+{{"demo": "LoginFinal.js"}}
```

Comment:
I have removed `hideToolbar": false` because it's already the default. I have added a check to avoid this case going forward.
I have removed `"bg": true` so it looks the same when you export to codesandbox https://codesandbox.io/s/m1r0yt?file=/demo.tsx/import it in your codebase.